Makefile

```
@@ -21,6 +21,25 @@ helm:
 
 dep-up:
 	CMD="dep up charts/sealed-secrets" $(MAKE) helm
+	$(MAKE) copy-crds
+
+copy-crds:
+	@echo "Copying CRDs from dependency chart to files directory..."
+	@mkdir -p charts/sealed-secrets/files
+	@cd charts/sealed-secrets/charts && \
+	if ls sealed-secrets-*.tgz > /dev/null 2>&1; then \
+		echo "Extracting dependency chart to access CRDs..."; \
+		tar -xzf sealed-secrets-*.tgz; \
+		if [ -d "sealed-secrets/crds" ]; then \
+			cp -r sealed-secrets/crds/* ../files/; \
+			echo "CRDs copied successfully"; \
+		else \
+			echo "Warning: CRDs directory not found in extracted dependency chart"; \
+		fi; \
+		rm -rf sealed-secrets; \
+	else \
+		echo "Warning: Dependency chart tgz file not found"; \
+	fi
```

Other code in this PR assumes a single CRD file `bitnami.com_sealedsecrets.yaml`, so the same can be done here.  Let `tar` handle the single file extraction to the correct place.  Also use `test` instead of `ls` to see if the archive exists.
```suggestion
	if [ -f sealed-secrets-*.tgz ]; then \
		echo "Extracting dependency chart CRDs..."; \
		if tar -xzf sealed-secrets-*.tgz -C ../files --strip-components 2 sealed-secrets/crds/bitnami.com_sealedsecrets.yaml; then \
			echo "CRDs copied successfully"; \
		else \
			echo "Warning: Failed to extract CRDs from dependency chart"; \
		fi; \
	else \
		echo "Warning: Dependency chart sealed-secrets-*.tgz file not found"; \
	fi
```